src/demo-app/a11y/button-toggle/button-toggle-a11y.html

```diff
@@ -0,0 +1,62 @@
+<section>
+  <h2>Single button toggle</h2>
+  <md-button-toggle>Yes</md-button-toggle>
+</section>
+
+<section>
+  <h2>Button toggles with icons</h2>
+  <md-button-toggle-group name="alignment">
```

Comment:
Without `role="group"`: when I moved focus to the multi-select button toggle group, the screen reader said "Flour, checkbox not checked".
With `role="group"`: when I moved the focus to the multi-select button toggle group, the screen reader said "Groceries group: Flour checkbox not checked"